### PR TITLE
docs(usage): add accessibility section

### PR DESCRIPTION
**Pull request by brandyscarney**
Sister PR: https://github.com/ionic-team/ionic-docs/pull/2807

Preview: https://ionicons-site-git-docs-accessibility-ionic1.vercel.app/ionicons/usage

### CODE REVIEW

File: src/components/usage-page/content.tsx [hunk 1]
```diff
@@ -132,6 +132,30 @@ export default function (version: string, name = "heart", suffix = "") {
...
+      <p>
+        Alternatively, if the icon is inside of another element that it is describing, that element
+        should have the <code>aria-label</code> added to it, and the icon should be hidden using
+        <code>aria-hidden</code>.
+      </p>
+      {highlight(
+        `<ion-button aria-label="Favorite">
+  <ion-icon name="heart" aria-hidden="true"></ion-icon>
+</ion-button>`
+      )}
```
Comment (liamdebeasi) [inline src/components/usage-page/content.tsx:157]: Will this get formatted correctly on the website?

**Review by liamdebeasi — APPROVED** [review]
One question, otherwise looks good

File: src/components/usage-page/content.tsx (hunk 1 above)
Comment (brandyscarney) [reply src/components/usage-page/content.tsx:157]: Yes, sorry I was going to include the link but vercel wasn't building but it looks like it just did: https://ionicons-site-git-docs-accessibility-ionic1.vercel.app/ionicons/usage

Comment (liamdebeasi) [reply src/components/usage-page/content.tsx:157]: perfect, thanks!

**Pull request merged**